fdbcli/TenantCommands.actor.cpp

```diff
@@ -93,6 +95,9 @@ void applyConfigurationToSpecialKeys(Reference<ITransaction> tr,
                                      TenantNameRef tenantName,
                                      std::map<Standalone<StringRef>, Optional<Value>> configuration) {
 	for (auto [configName, value] : configuration) {
+		if (configName == "assigned_cluster"_sr) {
+			throw invalid_option();
```

Comment:
We have another error that's a little more specific:

```suggestion
			throw invalid_tenant_configuration();
```

You could also add a trace event to aid in debugging if a user doesn't understand why they got this error.